lib/orbit/import/personnel.ex

```diff
@@ -43,12 +43,16 @@ defmodule Orbit.Import.Personnel do
           area: String.to_integer(&1[@area_value_field])
         }
       )
-      |> Enum.map(
-        &Repo.insert(&1,
-          on_conflict: {:replace_all_except, [:id]},
-          conflict_target: :badge_number
-        )
-      )
+      |> Enum.map(fn item ->
+        try do
+          Repo.insert(item,
+            on_conflict: {:replace_all_except, [:id]},
+            conflict_target: :badge_number
+          )
+        rescue
+          Postgrex.Error -> nil
```

Comment:
Do you expect this to happen?

Reply:
Yes; there's one preferred name that contains an invalid byte, so I'm skipping it for now

Reply:
ok! "catch all and ignore" scares me in general, but it's fine for the proof of concept.

Should we handle this case for real in Real Orbit? What's the name and the invalid byte issue? (DM me the name, don't put it here.) If it's a text encoding issue for a real name, we should see if we can get Orbit to handle it. If it's something silly like HR/CMS storing invalid UTF-8, then I guess we could filter it out before inserting.